statsmodels/regression/tests/test_lme.py

```diff
@@ -47,8 +49,7 @@ def __init__(self, meth, irfs, ds_ix):
         self.loglike = getattr(lme_r_results, "loglike" + bname)
 
         if hasattr(lme_r_results, "ranef_mean" + bname):
-            self.ranef_postmean = getattr(lme_r_results, "ranef_mean"
-                                          + bname)
+            self.ranef_postmean = getattr(lme_r_results, "ranef_mean" + bname)
```

Comment:
Is this going to cause flake8 complaints about lines longer than 80 characters?  Not a big deal, but if we're being picky...